lib/mix/lib/mix/tasks/test.ex

```diff
@@ -1,6 +1,22 @@
 defmodule Mix.Tasks.Test do
   defmodule Cover do
-    @moduledoc false
+    @default_formatters [:console, :html]
+    @default_threshold 90
+
+    @moduledoc """
```

Comment:
Let's please keep this module still private. I don't think we should expose it. Let's just have a section in the `mix test` docs that says the configurations for the default cover handling.